fonts/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License

```

Comment:
A note about the used fonts and their license(s) should also be included in the footer of our README.